docs/using-mycroft-ai/customizations/wake-word.md

```diff
@@ -167,11 +167,17 @@ mycroft-config reload
 
 ## Switching Wake Word Listeners
 
-To change the Wake Word Listener to PocketSphinx if it has been set to Precise, Speak:
+As per the [Technology Overview](https://mycroft-ai.gitbook.io/docs/mycroft-technologies/overview#wake-word-detection), there are two available listeners:
+1. Precise (default)
+2. PocketSphinx
 
-> Hey Mycroft, set the Listener to default
+Precise is likely appropriate for most users, however on legacy hardware PocketSphinx may be desired.
```

Comment:
open to maintainers including any/all/other reasons why someone might need to switch